<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadFileDialog</class>
 <widget class="QDialog" name="LoadFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>219</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Log file detected</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>A log file for today already exists.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Would you like to:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="addToExistingButton">
       <property name="text">
        <string>Add to default log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startNewButton">
       <property name="text">
        <string>Start a new log</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="browseForFileName">
     <property name="text">
      <string>Select a specific saved log</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="logTimeCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Log missing time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="trackMissingTimeRadio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Log missing time as &quot;Tracked&quot;</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="logTimeAsIgnoredRadio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Log missing time as &quot;Ignored&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="continueButton">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
